### ISSUE TITLE
Page Model Inheritance Breaks

### ISSUE THREAD

**Issue by dwasyl** [infoportugal/wagtail-modeltranslation#81]
I ran into a situation where I was inheriting a Page Model (that wasn't abstract) into another couple of models. When doing so, I would need to create a translation class for the source model (even if it was blank with no fields).

When I would try and create or save a page it would fail on with an error because it couldn't find the final page model in the list of fields (only the original).

For example:
BasePage - > FancyPage
                 -> FancierPage

Both would be actual models, and both would need to translation classes, but when saving a FancyPage page (or any other page) it would return an error saying "Cannot resolve keyword 'fancy_page' into field". In the list of fields that appears only the base_page is listed, not either of the page models that inherit from it.

**Comment by Alexandre27** [comment]
@dwasyl we think that problem was related with slug validation. That was changed in version 0.6, can you test it with this version?

**Comment by dwasyl** [comment]
@Alexandre27 I'm just testing 0.6 to see if it fixes all of the other problems I was having. I can add this to my list of things to test.

**Comment by dwasyl** [comment]
@Alexandre27 I tried this out, and the user has to manually set fields to empty (i.e. `fields = ()`) to override the work WagtailTranslationOptions does (since otherwise it tries to create duplicate fields). This is a reasonable thing to do since it's similar t modeltranslation, just probably good to document this somewhere. I also inherited options directly from `TranslationOptions` but that may not be necessary.
